dashboard/app/views/layouts/_header.html.haml

```diff
@@ -102,8 +102,7 @@
       lesson_extras_enabled = view_as.lesson_extras_enabled?(script)
     end
 
-    # don't trust outside content in parameter :puzzle_page - should be integer, so immediately call to_i
-    puzzle_page = params[:puzzle_page] ? params[:puzzle_page].to_i : ApplicationHelper::PUZZLE_PAGE_NONE
+    puzzle_page = params[:puzzle_page] ? params[:puzzle_page] : ApplicationHelper::PUZZLE_PAGE_NONE
```

Comment:
Is `ApplicationHelper::PUZZLE_PAGE_NONE` a string?

Reply:
no its an int, so this line doesn't need to change.